core/src/entry/mod.rs

```diff
@@ -40,6 +41,40 @@ impl CanPublish for EntryType {
         }
         true
     }
+
+    fn can_publish_from_state(&self, state: &State) -> bool {
```

Comment:
This is a bit of a hack for now. @lucksus any idea what the best way to call `can_publish` from inside `reduce_publish` where the context is not available?

Reply:
This seems perfectly fine to me. The `context` is mainly a means to make the state of an instance available outside of the reducers...